examples/ndcube_from_radio_images_example.py

```diff
@@ -0,0 +1,152 @@
+"""
+=================
+Creating an NDCube from solar radio interferometric images.
+=================
+
+How to make an NDCube from solar radio interferometric images.
+
+The example uses `dkist` https://docs.dkist.nso.edu/projects/python-tools/en/latest/ and `gwcs`
+to create a single wcs object that takes into account the changing crval of 
+solar radio images using `~dkist.wcs.models.VaryingCelestialTransform`. 
+The following example uses data from the NenuFAR radio telescope
+https://nenufar.obs-nancay.fr/en/homepage-en/
+"""
+import glob
+
+import numpy as np
+
+from multiprocessing import Pool
+
+from astropy.modeling import models
+from astropy import coordinates as coord
+from astropy.coordinates import SkyCoord, EarthLocation
+from astropy import units as u
+from astropy.io import fits
+from astropy.time import Time, TimeDelta
+from gwcs import wcs
+from gwcs import coordinate_frames as cf
+from ndcube import NDCube
+from sunpy.coordinates.frames import Helioprojective
+from sunpy.coordinates import sun
+
+from dkist.wcs.models import VaryingCelestialTransform
+
+##############################################################################
+# We first need to load some data. These exist on a machine somewhere and 
+# I don't know how to share them. 
+# Here we take every image at a single frequency
+
+ims = glob.glob("/data/mpearse/radio_images/SB158/*image.fits")
+ims.sort()
+
+def get_data(file):
+    with fits.open(file) as hdu:
+        header = hdu[0].header
+        data = np.squeeze(hdu[0].data)
+    
+    obstime = Time(header['date-obs']) # observational time
+    freq = header['crval3']*u.Hz # frequency of observation
+    wavelength = freq.to(u.m, equivalencies=u.spectral())
+    #data from Jy/beam to T_b
+    beam_semi_major_axis = 0.5*header['BMAJ']*u.deg #nothing in fits to suggest degrees, you just have to know
+    beam_semi_minor_axis = 0.5*header['BMIN']*u.deg
+    beam_area = ((np.pi*beam_semi_major_axis*beam_semi_minor_axis)/(4*np.log(2)))
+    data = data *(u.Jy/beam_area)
+    equiv = u.brightness_temperature(freq)
+    data = data.to(u.K, equivalencies=equiv)
+    
+    return data, obstime
+
+with Pool() as pool:
+    datatime = pool.map(get_data, ims)
+
+data = np.array([d[0] for d in datatime])
+time = Time(np.array([t[1] for t in datatime]))
+###########################################################################
+# The reference coordinate for radio observations typically changes as the
+# telescope pointing is updated to stay somewhat centred on the sun.
+# Thus, we will need create look up tables for the different reference 
+# coordinates and corresponding rotation matrices. We will also need the
+# location of the NenuFAR array.
+
+nenufar_ITRF = np.array((4323915,165533.67,4670321.7))
+array_loc = EarthLocation.from_geocentric(*nenufar_ITRF, u.m)
+
+def get_lookup_tables(file):
+    with fits.open(file) as hdu:
```

Comment:
Same comment as above.  Could these values be generated here rather than reading them in from a file.  I think the point of this example is to show how to create a WCS with a time-varying CRVAL rather than reading a FITS file.  Generating the data would make this easier to follow and remove the issue of sample data file handling in the package.

Reply:
Yeah, I've set it so that the data and lookup tables are just random numpy arrays of the correct shape. This also removes the need to use mulitproccessing and fits header parsing!